docs/references/contact-self.md

```diff
@@ -2,120 +2,156 @@
 
-GET / SET bot avatar
+获取/设置 机器人所使用账号的头像
 
-**Kind**: instance method of [`ContactSelf`](contact-self.md#ContactSelf)
+**类型**: [`ContactSelf`](contact-self.md#ContactSelf)类的实例方法
 
-| Param | Type |
+| 参数 | 类型 |
 | :--- | :--- |
 | \[file\] | `FileBox` |
 
-**Example** _\( GET the avatar for bot, return {Promise&lt;FileBox&gt;}\)_
+**示例** 
 
-```javascript
-// Save avatar to local file like `1-name.jpg`
+_\( 获取机器人账号的头像, 返回`FileBox`类型的对象\)_
 
-bot.on('login', async user => {
-  console.log(`user ${user} login`)
-  const file = await user.avatar()
-  const name = file.name
-  await file.toFile(name, true)
-  console.log(`Save bot avatar: ${user.name()} with avatar file: ${name}`)
-})
+```python
+# 保存头像到本地文件, 类似 `1-name.jpg`的格式
+import asyncio
+from wechaty import Wechaty, FileBox, Contact
+
+class MyBot(Wechaty):
+
+    async def on_login(self, contact: Contact) -> None:
+        print(f"用户{contact}登入")
+        file: FileBox = await contact.avatar()
+        name = file.name
+        await file.to_file(name, True)
+        print(f"保存头像: {contact.name} 和头像文件: {name}")
+
+
+asyncio.run(MyBot().start())
 ```
 
-**Example** _\(SET the avatar for a bot\)_
+**示例** 
+
+_\(设置机器人账号的头像\)_
+
+```python
+import asyncio
+from wechaty import Wechaty, FileBox, Contact
+
+class MyBot(Wechaty):
 
-```javascript
-import { FileBox }  from 'file-box'
-bot.on('login', user => {
-  console.log(`user ${user} login`)
-  const fileBox = FileBox.fromUrl('https://wechaty.github.io/wechaty/images/bot-qr-code.png')
-  await user.avatar(fileBox)
-  console.log(`Change bot avatar successfully!`)
-})
+    async def on_login(self, contact: Contact) -> None:
+        print(f"用户{contact}登入")
+        file_box: FileBox = FileBox.from_url('https://wechaty.github.io/wechaty/images/bot-qr-code.png')
+        await contact.avatar(file_box)
+        print(f"更改账号头像成功")
+
+
+asyncio.run(MyBot().start())
 ```
 
-### contactSelf.qrcode\(\) ⇒ `Promise <string>`
+### async def qr_code(self) ⇒ `str`
+
+获取机器人账号的二维码链接
 
-Get bot qrcode
+**类型**: [`ContactSelf`](contact-self.md#ContactSelf)的实例方法
 
-**Kind**: instance method of [`ContactSelf`](contact-self.md#ContactSelf)
+#### 示例
 
-#### Example
+```python
+import asyncio
+from wechaty import Wechaty
+from wechaty.user import ContactSelf
+from wechaty.utils.qrcode_terminal import qr_terminal_str
 
-```javascript
-import { generate } from 'qrcode-terminal'
-bot.on('login', async user => {
-  console.log(`user ${user} login`)
-  const qrcode = await user.qrcode()
-  console.log(`Following is the bot qrcode!`)
-  generate(qrcode, { small: true })
-})
+class MyBot(Wechaty):
+
+    async def on_login(self, contact: ContactSelf) -> None:
+        print(f"用户{contact}登入")
+        qr_code = await contact.qr_code()  # 获取二维码信息
+        print(qr_terminal_str(qr_code)) # 在控制台打印二维码
+
+asyncio.run(MyBot().start())
 ```
 
-### contactSelf.signature\(signature\) ⇒ `Promise <void>`
+### async def signature(self, signature: `str`) ⇒ `Any`
```

Comment:
The value of return should be None, why do you add `Any` at here?  from the 'stubfile' ?